db/patterns/google_beacon.eno

```diff
@@ -1,4 +1,4 @@
-name: Google Beacons
+name: Google Beacon
 category: site_analytics
 website_url: https://google.xyz
```

Comment:
(Sorry, not related to the PR, but since I see it):

I recall why we used the dummy page https://google.xyz while, of course, https://google.com would be more intuitive. It was a workaround for limitations in the Ghostery extension, but I think it's no longer needed.

I wonder if we should now change it also to https://google.com? @chrmod

Reply:
I would also prefer to change the url as those Beacons are present at google.com only